fedimint-testing/src/lib.rs

```diff
@@ -429,6 +445,25 @@ fn open_temp_db_and_copy(temp_path: String, src_dir: &Path) -> Database {
     Database::new(RocksDb::open(path).unwrap(), decoders)
 }
 
+pub fn open_temp_db(path: &Path) -> Database {
+    let decoders = ModuleDecoderRegistry::from_iter([
```

Comment:
Let's at least use a function where we hardcode it (if there isn't one already).

Reply:
Moved the decoders to be an argument instead of geneating them here